tests/mixins/test_taggable.py

```diff
@@ -17,22 +17,25 @@ def test_instantiates(self):
     def test_instantiates_with_tags(self):
         """ Ensures datafile inherits correctly from the Taggable class and passes arguments through
         """
-        tgd = MyTaggable(tags="")
-        self.assertEqual("", str(tgd.tags))
-        tgd = MyTaggable(tags=None)
-        self.assertEqual("", str(tgd.tags))
-        tgd = MyTaggable(tags="a b c")
-        self.assertEqual("a b c", str(tgd.tags))
+        taggable = MyTaggable(tags="")
+        self.assertEqual(len(taggable.tags), 0)
+
+        taggable = MyTaggable(tags=None)
+        self.assertEqual(len(taggable.tags), 0)
+
+        taggable = MyTaggable(tags="a b c")
+        self.assertEqual(set(taggable.tags), {"a", "b", "c"})
+
         with self.assertRaises(exceptions.InvalidTagException):
             MyTaggable(tags=":a b c")
 
     def test_instantiates_with_tag_group(self):
         """ Ensures datafile inherits correctly from the Taggable class and passes arguments through
         """
-        tgd = MyTaggable(tags="")
-        self.assertIsInstance(tgd.tags, TagGroup)
-        tgd2 = MyTaggable(tags=tgd.tags)
-        self.assertFalse(tgd is tgd2)
+        taggable_1 = MyTaggable(tags="")
+        self.assertIsInstance(taggable_1.tags, TagGroup)
+        taggable_2 = MyTaggable(tags=taggable_1.tags)
+        self.assertFalse(taggable_1 is taggable_2)
```

Comment:
If you're doing name refactoring in a pull request that's ok, but put a note in the PR, and... <see below>

Reply:
Even for local test variable names? I changed this set because I found it confusing whether something was a TagGroup or a Taggable. I'll update the rest